cmd/tpmk/sshclient.go

```diff
@@ -194,12 +197,30 @@ func runSSHClient(opt sshClientOptions, args []string) error {
 		HostKeyCallback: hostKeyCallback,
 	}
 
-	// Connect to the SSH server and start a session
-	client, err := ssh.Dial("tcp", host, config)
-	if err != nil {
-		return errors.Wrap(err, "connecting to "+host)
+	var client *ssh.Client
+	if opt.proxySocks5 != "" {
+		socksConnection, err := proxy.SOCKS5("tcp", opt.proxySocks5, nil, proxy.Direct)
+		if err != nil {
+			return errors.Wrap(err, "could not create socks5 proxy "+opt.proxySocks5)
+		}
+		conn, err := socksConnection.Dial("tcp", host)
+		if err != nil {
+			return errors.Wrap(err, "could not create connection with socks.")
+		}
```

Comment:
The `defer client.Close()` should in theory be closing the connection made here, but to be extra safe it'd be best to add a `defer conn.Close()` here as well. That's because if anything fails below, the connection won't be closed.